### PR TITLE
Update FINANCES.md

### PR DESCRIPTION
**Pull request by Frijol**
A couple of small updates to reflect current status. r? @johnnyman727 @HipsterBrown @rwaldron @tcr

### CODE REVIEW

File: FINANCES.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Finances
 
-The Tessel Project (as of May 2016) has its funds held by the non-profit Dojo Foundation.
+The Tessel Project (as of May 2016) has its funds held by non-profit foundations. The funds are currently held by the Bocoup Foundation.
```
Comment (HipsterBrown) [inline FINANCES.md]: Are the funds being help by multiple foundations? Or just the Bocoup Foundation now?

> The Tessel Project (as of May 2016) has its funds held by non-profit **foundations.**

Comment (Frijol) [inline FINANCES.md]: Just the Bocoup Foundation now, though there may still be some disentangling in process? @johnnyman727 can you comment?

Comment (johnnyman727) [inline FINANCES.md]: I think it's safe to say that we are officially disentangled and only part of the Bocoup Foundation.

**Comment by Frijol** [comment]
@johnnyman727 is this ok to merge?

File: FINANCES.md (hunk 1 above)
Comment (johnnyman727) [inline FINANCES.md]: @frojol I think this line is still confusing. Can it just use "foundation" instead of pluralized?

**Comment by Frijol** [comment]
@johnnyman727 is this better?

**Comment by johnnyman727** [comment]
Yeah LGTM

**Pull request merged**